DirectProgramming/DPC++FPGA/Tutorials/Features/floating_point_pragmas/README.md

```diff
@@ -0,0 +1,182 @@
+# Contraction and Reassociation on Floating Point Numbers
+This FPGA tutorial explains how to use the fp reassociate and fp contract pragmas for floating point numbers.
```

Comment:
Please add more detail here.  Here's what I recommend:

This FPGA tutorial explains how to use the fp reassociate and fp contract pragmas for floating point numbers.  These pragmas allow the compiler to change the order of certain floating point operations so that they map more efficiently to hardware.  The impact of this reordering is that the results may be altered slightly due to rounding that can occur after each floating point operation.